store/sdk/src/main/java/org/apache/carbondata/sdk/file/TestUtil.java

```diff
@@ -36,38 +36,34 @@
 import org.apache.avro.generic.GenericDatumReader;
 import org.apache.avro.generic.GenericDatumWriter;
 import org.apache.avro.io.DecoderFactory;
-import org.apache.avro.io.Encoder;
 import org.apache.avro.io.JsonDecoder;
 import org.apache.hadoop.conf.Configuration;
-import org.junit.Assert;
 
+@InterfaceAudience.Developer("Test")
 public class TestUtil {
 
-  public static Configuration configuration = new Configuration();
+  public static final Configuration configuration = new Configuration();
```

Comment:
Why is this test class changed?  How it is related to CLI tool

Reply:
Because now CLI is a separate module and testcase in CLI need to use TestUtil to write carbonfiles using SDK, so I have to move the TestUtil to src so that CLI testcase can use it. Is there other way?